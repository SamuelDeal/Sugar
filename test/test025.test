Int a = 5
while a < 10
	echo a++

a = 5
while a < 10
	echo ++a
