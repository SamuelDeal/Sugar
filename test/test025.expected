5
6
7
8
9
6
7
8
9
10
